x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/AutoFollowStats.java

```diff
@@ -73,24 +90,28 @@ public static AutoFollowStats fromXContent(final XContentParser parser) {
     private final long numberOfFailedRemoteClusterStateRequests;
     private final long numberOfSuccessfulFollowIndices;
     private final NavigableMap<String, ElasticsearchException> recentAutoFollowErrors;
+    private final NavigableMap<String, Long> trackingRemoteClusters;
```

Comment:
I wonder whether all stats should be reported per remote cluster, so `numberOfSuccessfulFollowIndices`, `recentAutoFollowErrors` etc. would then be reported on a per remote cluster bases. This makes more sense to me now considering how the auto follow coordinator currently operates.